NumberBaseball/Source.c

```diff
@@ -0,0 +1,150 @@
+#include <stdio.h>
+#include <stdlib.h> 
+#include<time.h> 
+
```

Comment:
put integers by define in parentheses

Reply:
Ah I see, missed that in the coding standard. Changed as needed.